viYarn/Chat/model/EpisodeContent.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  Chat.swift
+//  viYarn
+//
+//  Created by vieon on 25.09.2018.
+//  Copyright © 2018 RIEON. All rights reserved.
+//
+
+import Foundation
+import SwiftSoup
+
+struct EpisodeContent {
+    let chatMessages: [(author: String, messages: [String])]
+    let episodeDescription:String
+}
+
+extension EpisodeContent {
+    init? (html: String) {
+        guard let doc: Document = try? SwiftSoup.parse(html) else { return nil }
+        guard let elements = try? doc.getAllElements() else { return nil }
+        
+        guard let episodeDescriptionElement = try? doc.select("p").first() else { return nil }
+        var episodeDescription = ""
+        if let elementDesc = episodeDescriptionElement {
```

Comment:
p0: i think we have to refactor this part. It hast way too many nested if statements.